loop/views.py

```diff
@@ -51,19 +51,20 @@ def dashboard(request):
     return render(request, 'app_dashboards/loop_dashboard.html')
 
 
-def second_loop_page(request):
-    return render(request, 'app_dashboards/second_loop_page.html')
-
-
 def filter_data(request):
-    aggregators = LoopUser.objects.extra(select={'name':'name_en'}).values('user__id', 'name')
-    villages = Village.objects.extra(select={'village_name':'village_name_en'}).values('id', 'village_name')
-    crops = Crop.objects.extra(select={'crop_name':'crop_name_en'}).values('id', 'crop_name')
-    mandis = Mandi.objects.extra(select={'mandi_name':'mandi_name_en'}).values('id', 'mandi_name')
-    gaddidars = Gaddidar.objects.extra(select={'gaddidar_name':'gaddidar_name_en'}).values('id', 'gaddidar_name')
+    language = request.GET.get('language')
+    aggregators = LoopUser.objects.all().values('user__id', 'name', 'name_en')
+    villages = Village.objects.all().values('id', 'village_name', 'village_name_en')
+    crops = Crop.objects.all().values('id', 'crop_name')
+    cropsLang = Croplanguage.objects.values('crop__id', 'crop_name')
+    cropslanguage = [{'id': x['crop__id'],
```

Comment:
follow one convention either camelcase or underscore but make it crops_language at this place and wherever referred.